Makefile.checker

```diff
@@ -22,7 +22,7 @@ CHICKEN:=bin/coqchk$(EXE)
 
 # The sources
 
-CHKLIBS:= -I config -I clib -I lib -I checker
+CHKLIBS:= -I config -I clib -I lib -I checker -I kernel -I kernel/byterun -I library -I engine
```

Comment:
There should be no need to `-I library -I engine` now.

Reply:
Right, it was a leftover, now removed.